sympy/vector/parametricregion.py

```diff
@@ -82,3 +87,84 @@ def parameters(self):
     @property
     def dimensions(self):
         return len(self.limits)
+
+
+@singledispatch
+def parametric_region(reg):
```

Comment:
Now that it returns a list... maybe it can be renamed to something like `parametric_region_list` or `get_parametric_region_list` or `_get_parametric_region_list`.

Reply:
I have changed it to `parametruc_region_list` for now. I think this function can also be used outside the `vector_integrate` so its name should not start with `_`. 

We can also rename it to `parametric_regions`?

Reply:
Yes.